### ISSUE TITLE
Wi-Fi icon grayed out even when active

### ISSUE THREAD

**Issue by ferike1** [snwh/paper-icon-theme#579]
I'm using the Paper icon pack with Solus OS. One little bug I've noticed is that when selecting this icon pack, the Wi-Fi icon is grayed out, even when Wi-Fi is on and active. There are no problems with the default Arc icon theme though. Also, there were no problems in Ubuntu Budgie with the same icon pack, which is strange.

You can see the problem in this picture:
http://i.imgur.com/eT7nvMV.png

The left one is with Paper, and the right one is with Arc.

I was suggested to raise an issue here in the Solus forums:
https://solus-project.com/forums/viewtopic.php?f=14&t=8196#p24781

**Comment by snwh** [comment]
I'm not sure why that is the case

**Comment by ferike1** [comment]
But do you think it is related to Solus or the icon pack itself?

**Comment by snwh** [comment]
Couldn't tell ya. Unless the network applet isn't using the symbolic icons.

**Comment by Justinzobel** [comment]
@ikeydoherty can you please jump in and assist here?

**Comment by ikeydoherty** [comment]
This is actually down to GTK theme stuffs, which was fixed in Adapta at some point. So, make sure you have latest versions of All The Things. There is also about to be a new Budgie release so that should include any latent fixes.

**Comment by ferike1** [comment]
I have installed it through Software Center, so it should be the latest version, I suppose it also gets updated. I have version 3.90.0.166-48 of adapta-gtk-theme. I have also applied the latest updates to the Budgie desktop last week.

**Comment by ferike1** [comment]
I have noticed something strange. When there's another icon next to the wi-fi, then the wi-fi icon also displays correctly. I realized it when I opened VLC. Then even if I close VLC, the icon stays white as it should, but on a reboot, it will get greyed out again. Hopefully this information will be helpful.

![image](https://user-images.githubusercontent.com/29072315/27773164-f4ecc650-5f73-11e7-8120-d0e3e7a18bcd.png)
